File: public/phase-2-20160620022203/C/add-and-remove.c
// Name : Add and Remove
#include <stdio.h>
int main() {
	// input:
	// n : size of array
	// a : source array
	int n, a[10000], maximum_number_of_equal_elements=1, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `maximum_number_of_equal_elements`

	// output
	printf("%d", maximum_number_of_equal_elements);
	return 0;
}

File: public/phase-2-20160620022203/JAVASCRIPT/add-and-remove.js
// Name : Add and Remove
// input:
// n : size of array
// a : source array
var index,maximum_number_of_equal_elements;
var n = parseInt(readline(), 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<n;index++)
	a.push(parseInt(a_elements[index], 10));


// write your code here
// store your results in `maximum_number_of_equal_elements`

// Dummy Data
maximum_number_of_equal_elements = 1;


// output
print(maximum_number_of_equal_elements);
